Kudu.Services/FetchHelpers/OneDriveHelper.cs

```diff
@@ -62,7 +62,7 @@ internal ILogger Logger
             ChangeSet changeSet = null;
             string cursor = _settings.GetValue(CursorKey);
             ChangesResult changes = null;
-            using (_tracer.Step("Getting delta changes with cursor: {0} ...", cursor))
+            using (_tracer.Step("Getting delta changes with cursor: {0}...", cursor.Truncate(5)))
```

Comment:
That OneDrive __cursor__ thing is VERY long and breaks the UI like this:

![image](https://cloud.githubusercontent.com/assets/6472374/15979696/56d7e3a4-2f6e-11e6-9e89-b5bb5bcfc314.png)